### PR TITLE
doc: Mention the missing reflected operations

### PR DESCRIPTION
**Pull request by paulofreitas**
Just a minor doc update to maintain consistency with other reflected operations documented in `reference/expressions`. 👍 

----
📚 Documentation preview 📚: https://cpython-previews--119931.org.readthedocs.build/

### CODE REVIEW

**Review by Eclips4 — APPROVED** [review]
Thanks!

File: Doc/reference/expressions.rst [hunk 1]
```diff
@@ -1367,7 +1367,7 @@ This operation can be customized using the special :meth:`~object.__add__` and
 The ``-`` (subtraction) operator yields the difference of its arguments.  The
 numeric arguments are first converted to a common type.
 
-This operation can be customized using the special :meth:`~object.__sub__` method.
+This operation can be customized using the special :meth:`~object.__sub__` and :meth:`~object.__rsub__` methods.
```
Comment (mdickinson) [inline Doc/reference/expressions.rst]: Nit: should we wrap this line? It's a little long.

Comment (mdickinson) [inline Doc/reference/expressions.rst]: For convenience, here's the suggested wrapping in the form of an actual suggestion:

```suggestion
This operation can be customized using the special :meth:`~object.__sub__`
and :meth:`~object.__rsub__` methods.
```

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]
There is also the same issue for `__rpow__`, `__rtruediv__`, `__rfloordiv__`, `__rmod__`, `__rlshift__`, `__rrshift__`, and maybe others.

It is better to fix them all in a single PR.

Alternatively, we can remove references to `__radd__` and `__rmul__`.

**Comment by paulofreitas** [comment]
> There is also the same issue for `__rpow__`, `__rtruediv__`, `__rfloordiv__`, `__rmod__`, `__rlshift__`, `__rrshift__`, and maybe others.
> 
> It is better to fix them all in a single PR.
> 
> Alternatively, we can remove references to `__radd__` and `__rmul__`.

Note that `__rand__`, `__rxor__` and `__ror__` were also documented. I went with the first suggestion and added the other missing reflected operations. 👍

**Review by serhiy-storchaka — COMMENTED** [review]
For divisions and shifts, I think that it is better to be explicit what dunder customizes what operator.

While we are here, I think it is worth to document also the dunders for the `@` operator.

**Comment by paulofreitas** [comment]
> For divisions and shifts, I think that it is better to be explicit what dunder customizes what operator.
> 
> While we are here, I think it is worth to document also the dunders for the `@` operator.

Hope I covered both things as intended.  👍

File: Doc/reference/expressions.rst [hunk 2]
```diff
@@ -1314,8 +1318,9 @@ integer; the result is that of mathematical division with the 'floor' function
 applied to the result.  Division by zero raises the :exc:`ZeroDivisionError`
 exception.
 
-This operation can be customized using the special :meth:`~object.__truediv__` and
-:meth:`~object.__floordiv__` methods.
+This operation can be customized using the special :meth:`~object.__truediv__`,
```
Comment (serhiy-storchaka) [inline Doc/reference/expressions.rst]: There are two different operations.

File: Doc/reference/expressions.rst [hunk 3]
```diff
@@ -1388,8 +1395,11 @@ The shifting operations have lower priority than the arithmetic operations:
 These operators accept integers as arguments.  They shift the first argument to
 the left or right by the number of bits given by the second argument.
 
-This operation can be customized using the special :meth:`~object.__lshift__` and
-:meth:`~object.__rshift__` methods.
+The left shift operation can be customized using the special :meth:`~object.__lshift__`
+and :meth:`~object.__rlshift__` methods.
+
```
Comment (serhiy-storchaka) [inline Doc/reference/expressions.rst]: Would not be better to merge these two paragraphs?
```suggestion
```

File: Doc/reference/expressions.rst (hunk 2 above)
Comment (paulofreitas) [reply Doc/reference/expressions.rst]: > There are two different operations.

Nice catch, I was forgetting this one. 👍

File: Doc/reference/expressions.rst (hunk 3 above)
Comment (paulofreitas) [reply Doc/reference/expressions.rst]: For sure, merged it and did the same formatting for division and floor division operations. 👍

**Review by serhiy-storchaka — APPROVED** [review]
LGTM.

Thank you for your contribution @paulofreitas.

**Comment by paulofreitas** [comment]
> LGTM.
> 
> Thank you for your contribution @paulofreitas.

Thank you for all your help reviewing! Glad to contribute here, looking forward to contribute more!  🎉 🐍

**Comment by serhiy-storchaka** [comment]
Looking forward to review more your PRs.

**Pull request merged**